pom.xml

```diff
@@ -208,7 +208,7 @@
             <properties>
                 <scala.binary.version>2.12</scala.binary.version>
                 <scala.version>2.12.11</scala.version>
-                <spark.version>3.1.2</spark.version>
+                <spark.version>3.3.0</spark.version>
```

Comment:
Could you remove the spark24 and spark31 profiles in pom file?

Reply:
as in update the name of the profile, or add a new one? i renamed the profile, let me know if I should add a second one for Spark 3.3.0